src/cascadia/CascadiaPackage/Package-Dev.appxmanifest

```diff
@@ -16,7 +16,7 @@
 
   <Identity
     Name="WindowsTerminalDev"
-    Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US"
+    Publisher="CN=lhecker"

```

Comment:
LOL wut...

Reply:
Curiously this happens automatically when you go to "Publish > Create App Packages...".